goosebit/updates/swdesc/rauc.py

```diff
@@ -0,0 +1,42 @@
+import configparser
+import logging
+
+import semver
+from anyio import Path, open_file
+from PySquashfsImage import SquashFsImage
+
+from .func import sha1_hash_file
+
+MAGIC = b"hsqs"
+
+logger = logging.getLogger(__name__)
+
+
+async def parse_file(file: Path):
```

Comment:
There should be at least one test for this parsing code (as there is for swu images)

Reply:
Added one, it doesnt do the compatibility parsing, see https://github.com/UpstreamDataInc/goosebit/pull/132#discussion_r1754789482